src/main/resources/application.conf

```diff
@@ -99,6 +99,21 @@ source {
       password = ${?SOURCE_BITBUCKET_SERVER_AUTH_PASSWORD}
     }
   }
+  bitbucket-cloud {
+    api {
+      url = ${?SOURCE_BITBUCKET_CLOUD_API_URL}
+    }
+
+   organization = ${?SOURCE_BITBUCKET_CLOUD_ORGANIZATION}
```

Comment:
it seems you're requiring this to be nonempty in your code. In which case, please change `${?...}` to `${...}` where no default value is given